Tasks/AzureDevOpsTestPlanV0/Invokers/pythonivoker.ts

```diff
@@ -0,0 +1,27 @@
+import { spawn } from '../testexecutor'
+import tl = require('azure-pipelines-task-lib/task');
+
+export async function executepythontests(testsToBeExecuted: string[]) {
+
```

Comment:
Can we add a comment to depict the executable command, we are trying to form?

Reply:
added comments to add sample executable command in framework invokers